src/app/auth/login/login.component.ts

```diff
@@ -1,16 +1,30 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { UserService } from '../services/user.service';
+import { AuthService } from '../services/auth.service';
+import { ModalDirective } from 'ngx-bootstrap/modal';
 
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss'],
 })
-export class LoginComponent implements OnInit {
+export class LoginComponent {
+  public massage: string;
+  @ViewChild('modal', {static: false}) modal: ModalDirective;
+  frmlogin = new FormGroup({
```

Comment:
make it a full word, second word starts with uppercase
```suggestion
  formLogin = new FormGroup({
```